.travis.yml

```diff
@@ -10,16 +9,37 @@ php:
   - hhvm
  
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: SYMFONY_VERSION='2.3.*' FOSUSERBUNDLE_VERSION='~1.3'
```

Comment:
Single quote is not necessary.

And I think separate symfony version tests and fos user version tests would be clearer.